Gem/Nodes/ViewModels/AddNodeSceneViewModel.swift

```diff
@@ -63,39 +75,35 @@ extension AddNodeSceneViewModel {
         let provider = ChainServiceFactory(nodeProvider: CustomNodeULRFetchable(url: url))
         let service = provider.service(for: chain)
 
-        async let isNodeSync = service.getInSync()
-        async let chainId = service.getChainID()
-        async let blockNumber = service.getLatestBlock()
-
         do {
-            let (isSynced, networkId, blockNumber) = try await (isNodeSync, chainId, blockNumber)
+            let (latency, chainId) = try await fetchChainID(service: service)
```

Comment:
those methods should be all async in parallel, currently it's split in two, twice slower

Reply:
The idea why we doing that way - to receive firstly chainId, compare  it with chainId we looking for and if chainId is different, do not  use 2 more unneeded requests. The  latency value we are getting with a chainId also. If we do a 3 async let we are not receive proper latency value and in case of not correct chain id, we do 2 more unneeded request to a node.

Reply:
it does not matter if we receive a latency or not. it needs to run fast, run 3 fetch requests and parallel and present results to the user